### ISSUE TITLE
[Bug Report] New render API not working

### ISSUE THREAD

**Issue by trigaten** [openai/gym#2825]
From [this documentation page](https://www.gymlibrary.ml/content/api/), running the following yields an error.

```python
import gym
env = gym.make("LunarLander-v2", render_mode="human")
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/anaconda3/envs/vid/lib/python3.9/site-packages/gym/envs/registration.py", line 503, in make
    env = cls(**_kwargs)
TypeError: __init__() got an unexpected keyword argument 'render_mode'
```

**Comment by pseudo-rnd-thoughts** [comment]
That documentation update seems a bit early, that PR is still in progress and will be merged soon https://github.com/openai/gym/pull/2671
For now, please remove the `render_mode` parameter

**Comment by RedTachyon** [comment]
Do we have any proper versioning of the documentation repo? It seems the built version is `master`, and new updates are also pushed to `master`, which isn't necessarily a great idea. We should probably do it like most other libraries, where you have a docs page for the last stable release, and separate docs for `master`

**Comment by younik** [comment]
Hello @trigaten,
https://github.com/Farama-Foundation/gym-docs/pull/124 was the pull to update documentation for https://github.com/openai/gym/pull/2671; the updated snippets will work when https://github.com/openai/gym/pull/2671 is merged.
Sorry for not pointing this out clearly. It would be nice to have a mechanism as @RedTachyon pointed out or you can reverse https://github.com/Farama-Foundation/gym-docs/pull/124 and merge it only after https://github.com/openai/gym/pull/2671
Thanks

**Comment by trigaten** [comment]
@RedTachyon @younik How exactly should versioned documentation work?
Would we have a branch for each new gym version and have the site build from the most up to date branch?
When would master be pushed to?

**Comment by RedTachyon** [comment]
So I'm not actually sure what the industry standard is of doing this, but the way I'd envision it is:
- Branch "stable" for the latest release (and the default build target for the website)
- Branch "master" for incremental updates (and maybe as a build target under a subdomain or under a specific path, so that we can access it, but it's not the default that people see)
- When we update the main library from version X to Y, the current "stable" gets pushed to a new branch named vX, and current master gets pushed to "stable"

**Comment by AhmedTechOffical** [comment]
Hi, I got a similar problem but instead, its saying that I need to put render_mode = human and I did, yet its telling me I need to do that, Any help please?
![image](https://user-images.githubusercontent.com/41643485/172152701-6c4c6664-7702-4109-980a-54069a7fd759.png)

**Comment by DDDOH** [comment]
Before the released package can catch up with the current document, any instructions on how to use the old render API? Or when will a new package will get published to match the current document?

**Comment by younik** [comment]
> Before the released package can catch up with the current document, any instructions on how to use the old render API? Or when will a new package will get published to match the current document?
> 
> or maybe I'll try building from source before the release of a new version

Hello, if you want to use the old API and you don't know how it works, you can follow old tutorials, like [this one](https://blog.paperspace.com/getting-started-with-openai-gym/)

**Comment by DDDOH** [comment]
Thanks a lot, that's a good one! Looking forward to the brand new API anyway!

**Comment by BlueskyFR** [comment]
May I ask why this issue was closed? The case where someone uses an other version than the master branch (so more than 99% of the cases) is still not handled AFAIK

**Comment by DDDOH** [comment]
It's mainly caused by document and code unmatching. be a little bit patient bro.
